administrator/components/com_media/resources/scripts/store/state.js

```diff
@@ -23,6 +23,4 @@ export default {
     selectedDirectory: options.providers[0].name + ':/',
     // The currently selected items
     selectedItems: [],
-    // The currently selected items
-    selectedItems: [],
```

Comment:
I would do that in another pr against the milestone 1 branch. One thing in one pr.